src/backend/storeManagers/hyperplay/games.ts

```diff
@@ -157,6 +161,36 @@ export async function pause(appName: string): Promise<void> {
   dl.DownloadItem.pause()
 }
 
+// check for valid json file inside the folder before importing
+// if none was found, check the first folder inside the folder
+// return the path to the folder with a valid json file
+// if none was found return empty string
+// this is necessary because we do not know which folder the user has selected, the main folder or the game folder
+const isValidGameFolder = (appName: string, folderPath: string): string => {
```

Comment:
this is nit pick and nonblocker: I find it weird that this function doesn't return a boolean even though it uses a `is__` prefix.

Reply:
I agree. Seems `getValidGameFolderPath` is more accurate

Reply:
Good point.
I think that was the idea but then I decided to return the folder path instead.
I will rename it.